daml/Tests/Distribution/Syndication/Issuance.daml

```diff
@@ -92,28 +94,42 @@ issuance Parties{..} Origination{..} = do
     dateOfTrade = dateOfSettlement
 
   -- Step 1
+  now <- getTime
+  let
+    secret = "mysecret"
+    hashlock = sha256 secret
+    expiry = addRelTime now $ hours 1
   dvpCid11 <- submit operator do exerciseCmd tranche1Cid Structuring.InstructIssuerSettlement with price; dateOfSettlement; dateOfTrade
   dvpCid12 <- submit operator do exerciseCmd tranche2Cid Structuring.InstructIssuerSettlement with price; dateOfSettlement; dateOfTrade
-  dvpCid13 <- submit operator do exerciseCmd tranche3Cid Structuring.InstructIssuerSettlement with price; dateOfSettlement; dateOfTrade
+  dvpCid13 <- submit operator do exerciseCmd tranche3Cid Structuring.InstructIssuerSettlementHtlc with price; dateOfSettlement; dateOfTrade; hashlock; expiry
   allocateInstructions [ issuer, custodian2, settlementBank ]
+  allocateInstructionsHtlc [ issuer, custodian2 ]
   signInstructions [ issuer, custodian2, settlementBank ]
-  settleTrades bondRegistrar [ dvpCid11, dvpCid12, dvpCid13 ]
+  signInstructionsHtlc [ custodian2, settlementBank ]
+  settleTrades bondRegistrar [ dvpCid11, dvpCid12 ]
+  settleTradeHtlc settlementBank (secret, dvpCid13)
 
   -- Step 2
   dvpCid21 <- submit operator do exerciseCmd tranche1Cid Structuring.InstructBndSettlement with price; dateOfSettlement; dateOfTrade
   dvpCid22 <- submit operator do exerciseCmd tranche2Cid Structuring.InstructBndSettlement with price; dateOfSettlement; dateOfTrade
-  dvpCid23 <- submit operator do exerciseCmd tranche3Cid Structuring.InstructBndSettlement with price; dateOfSettlement; dateOfTrade
+  dvpCid23 <- submit operator do exerciseCmd tranche3Cid Structuring.InstructBndSettlementHtlc with price; dateOfSettlement; dateOfTrade; hashlock; expiry
```

Comment:
Should use a different secret / hashlock here - the two settlements are independent

Reply:
yes, you are right, I was such a lazy person, updated.